lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
```

Comment:
I think the only way to get skipCapabilities to work properly is going to be to pass in a pin map when a new board instance is created. You'd have to create a pin map for each unique architecture.